appengine-java21/bigtable/README.md

```diff
@@ -1,13 +1,13 @@
-Bigtable-hello-j8
+Bigtable-hello-j21
 =================
 
-<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=appengine-java8/bigtable/README.md">
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=appengine-java21/bigtable/README.md">
 <img alt="Open in Cloud Shell" src ="http://gstatic.com/cloudssh/images/open-btn.png"></a>
 
-Moves the Bigtable Hello World application to Google App Engine Standard for Java 8.
+Moves the Bigtable Hello World application to Google App Engine Standard for Java 21.
 
 
-* [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+* [Java 21](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
 * [Maven](https://maven.apache.org/download.cgi) (at least 3.3.9)
 * [Gradle](https://gradle.org)
 * [Google Cloud SDK](https://cloud.google.com/sdk/) (aka gcloud)
```

Comment:
I know this is preexisting but Google Cloud SDK and Google Cloud CLI (gcloud) are different things

Reply:
Changed all Cloud SDK to Cloud CLI.